test/e2e/tsconfig.json

```diff
@@ -8,7 +8,8 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "target": "ES2015",
-    "types": ["node", "jasmine", "@wdio/globals/types", "ajv"]
+    "types": ["node", "jasmine", "@wdio/globals/types", "ajv"],
+    "allowJs": true
```

Comment:
It is, because:
* e2e tests are importing `@datadog/browser-rum/test`
* `packages/rum/test/index.ts` is importing `mockWorker`
*  `mockWorker` now imports `string2buf` from `deflate.js`
* `deflate.js` is using `export` statements which are not supported in nodejs for .js files.

So we need tsc to transpile JS files as well. Moving string2buf in a TS file could also work, but I prefered to change the TS config because we might want to keep pako sources as close to the original as possible to ease its upgrade, but this could be challenged of course.